### DISCUSSION TITLE
Problem with `:class` in Vue

### DISCUSSION THREAD

**Discussion by amir20** [unocss/unocss#2315] [General]
I am wondering if anybody else has ran to a similar issue. I would expect it to be a major bug if reproducible. 

I have noticed that `<aside :class="{ 'fixed': isMobile }">` works. However, when removing the quotes `<aside :class="{ fixed: isMobile }">` it does not. I am using @antfu's es-lint which removes the quotes after save. 

Note, that this is inconsistent. If I added the quotes without doing `eslint --fix` it works and after that it continues to work. On a cold boot, it does not. 

For now, I am using `<aside :class="isMobile ? 'fixed' : ''">`. 

Is this a bug?

**Comment by antfu** [comment]
Yes, it sounds like a bug.
https://uno.antfu.me/play/?html=DwQwzglgJgpgBALgMYBtxgLwCIDecDkAZhAB4xT4JwRgCyA9gEYQrwC%2BWAfAFCiSyJU6bHhCMw9FAFcALjCo0GzVnA6cgA&config=JYWwDg9gTgLgBAbwFBzgEwKYDNgDsMDCEuOA5gDQpxhQYDOGMAqrhJQL5xZQQhwDkAV1YBjOnX5IkGAB6RY6bAENBAG3iYc%2BIiWCkAFMlRQ19AFxwA2lVSWA9CdUY75OPoCUcALwA%2BNwjhHDAsARjh2dwBdSlRbB1MXN09ff0DTCwAmcKjXOzs4GAALYDo4ErgQJRgRQow0KkikCKQgA&css=Q&options=N4IgLgTghgdgzgMwPYQLYgFwKgGzgUwF8g

/cc @sibbng maybe it's a regression of the recent regex change?

**Comment by amir20** [comment]
Why didn't I think of Uno Playground :) 

Thanks for confirming. Should I file a bug for now? Can discussions be converted to issues?
